RoddyCore/src/de/dkfz/roddy/execution/io/ExecutionService.groovy

```diff
@@ -103,7 +103,7 @@ abstract class ExecutionService implements BEExecutionService {
         ClassLoader classLoader = LibrariesFactory.getGroovyClassLoader()
 
         RunMode runMode = Roddy.getRunMode()
-        String executionServiceClassID = Roddy.getApplicationProperty(runMode, Constants.APP_PROPERTY_EXECUTION_SERVICE_CLASS, SSHExecutionService.class.getName())
+        String executionServiceClassID = Roddy.getOrSetApplicationProperty(runMode, Constants.APP_PROPERTY_EXECUTION_SERVICE_CLASS, SSHExecutionService.class.getName())
```

Comment:
Seeing all these Roddy. accesses makes me whonder, if we can just initialize the execution services differently. This would lead to less complexity.

Reply:
Maybe again the `@Builder` annotation to generate a parameter object?